spec/controllers/tokens_controller_spec.rb

```diff
@@ -31,6 +31,34 @@
     end
   end
 
+  describe 'when there is a failure due to a custom error' do
+    it 'returns the error response with a custom message' do
+      # NOTE: I18n will look for this key `en.doorkeeper.errors.messages.my_message` in a locale file
+      allow(I18n).to receive(:translate)
+        .with("my_message", {:scope=>[:doorkeeper, :errors, :messages]})
```

Comment:
Redundant curly braces around a hash parameter.<br>Place the . on the previous line, together with the method call receiver.<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.